### ISSUE TITLE
Feature corrections

### ISSUE THREAD

**Issue by argallmr** [colinrsmall/mp-dl-unh#7]
Some of the features are calculated or read from file incorrectly. I know you said you got some of these already, but I am going to put them all here anyway. Feel free to close it immediately.

### DES
Missing columns from `energyspectr_omni`, `bulkv`, `heatq`. Namely, `energyspectr_omni_31`, 'bulkv_2` or `heatq_2`.

```
if fpi_dis_cdf[var][...].ndim > 1:  # Data includes more than one measurement
    for i in range(0, fpi_dis_cdf[var].shape[1] - 1):
        fpi_dis_df[var + f'_{i}'] = fpi_dis_cdf[var][:, i]
```

should be (no `- 1`)

```
if fpi_dis_cdf[var][...].ndim > 1:  # Data includes more than one measurement
    for i in range(0, fpi_dis_cdf[var].shape[1]):
        fpi_dis_df[var + f'_{i}'] = fpi_dis_cdf[var][:, i]
```

### DIS
Same problem as for DES

### AFG
`Bz_Q` was calculated using `Bx`

### Multiple Instruments
* `plasma_beta` should be `(Pi + Pe) / P_B`, where `Pi`, `Pe`, and `P_B` are the (scalar) pressures from DIS, DES, and AFG.
* `temperature_ratio` should be `Ti/Te`, where `Ti` and `Te` are the scalar temperatures from DIS and DES

**Comment by colinrsmall** [comment]
Fixing this in the notebook version of sitl_downloader (https://colab.research.google.com/drive/1SipPAXb3-PXpWHRYkqu5zYjkU4GHiaNU). Will copy over to the code here once everything's fixed.

**Comment by argallmr** [comment]
Found another one:

```
for var in fpi_dis_var_list_3d:
            fpi_dis_df[f'{var}_x1_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 0, 0]
            fpi_dis_df[f'{var}_x2_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 1, 0]
            fpi_dis_df[f'{var}_x2_y2'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 1, 1]
            fpi_dis_df[f'{var}_x3_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 0]
            fpi_dis_df[f'{var}_x3_y2'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 1]
            fpi_dis_df[f'{var}_x3_y3'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 2]
```

should be

```
for var in fpi_dis_var_list_3d:
            fpi_dis_df[f'{var}_x1_y1'] = fpi_dis_cdf[f'{var}'][:, 0, 0]
            fpi_dis_df[f'{var}_x2_y1'] = fpi_dis_cdf[f'{var}'][:, 1, 0]
            fpi_dis_df[f'{var}_x2_y2'] = fpi_dis_cdf[f'{var}'][:, 1, 1]
            fpi_dis_df[f'{var}_x3_y1'] = fpi_dis_cdf[f'{var}'][:, 2, 0]
            fpi_dis_df[f'{var}_x3_y2'] = fpi_dis_cdf[f'{var}'][:, 2, 1]
            fpi_dis_df[f'{var}_x3_y3'] = fpi_dis_cdf[f'{var}'][:, 2, 2]
```

The result is that the temperature tensor variable contains the same data as the pressure tensor (those are the only two 3d variables).

**Comment by colinrsmall** [comment]
Good catch! Apologies for the sloppy code...On Jun 2, 2020 8:30 PM, Matthew Argall <notifications@github.com> wrote:
Found another one:
for var in fpi_dis_var_list_3d:
            fpi_dis_df[f'{var}_x1_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 0, 0]
            fpi_dis_df[f'{var}_x2_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 1, 0]
            fpi_dis_df[f'{var}_x2_y2'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 1, 1]
            fpi_dis_df[f'{var}_x3_y1'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 0]
            fpi_dis_df[f'{var}_x3_y2'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 1]
            fpi_dis_df[f'{var}_x3_y3'] = fpi_dis_cdf[f'{spacecraft}_dis_prestensor_dbcs_fast'][:, 2, 2]

should be
for var in fpi_dis_var_list_3d:
            fpi_dis_df[f'{var}_x1_y1'] = fpi_dis_cdf[f'{var}'][:, 0, 0]
            fpi_dis_df[f'{var}_x2_y1'] = fpi_dis_cdf[f'{var}'][:, 1, 0]
            fpi_dis_df[f'{var}_x2_y2'] = fpi_dis_cdf[f'{var}'][:, 1, 1]
            fpi_dis_df[f'{var}_x3_y1'] = fpi_dis_cdf[f'{var}'][:, 2, 0]
            fpi_dis_df[f'{var}_x3_y2'] = fpi_dis_cdf[f'{var}'][:, 2, 1]
            fpi_dis_df[f'{var}_x3_y3'] = fpi_dis_cdf[f'{var}'][:, 2, 2]

The result is that the temperature tensor variable contains the same data as the pressure tensor (those are the only two 3d variables).
